index.html

```diff
@@ -8,7 +8,178 @@
     <title>PricingPage</title>
     <link rel="preconnect" href="https://fonts.googleapis.com" />
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
-    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@500;600;700&display=swap"
-      rel="stylesheet"/>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@500;600;700&display=swap"
+      rel="stylesheet"
+    />
   </head>
+
+  <body>
+    <header>
+      <button class="btn-back">
+        <img class="img" src="img/Arrow Forward.svg" />Back
+      </button>
+      <h1 class="title">Pricing</h1>
+      <div class="subtitle">You can update or downgrage at any time</div>
+    </header>
+
+    <div class="switch-container">
+      <span class="monthly">Monthly</span>
+      <img class="img-switch" src="/img/Switch.svg" />
```

Comment:
in src missed ./
```suggestion
      <img class="img-switch" src="./img/Switch.svg" />
```
without it user can not see element
or use can left without /

Reply:
by the way, looks like you can do it like an element, not like img )